scanner/src/main/kotlin/experimental/ScannerWrapper.kt

```diff
@@ -36,6 +38,15 @@ import org.ossreviewtoolkit.scanner.ScannerCriteria
  * The base interface for all types of scanners.
  */
 sealed interface ScannerWrapper {
+    companion object {
+        private val LOADER = ServiceLoader.load(ScannerWrapperFactory::class.java)!!
+
+        /**
+         * The list of all available scanners in the classpath.
+         */
+        val ALL by lazy { LOADER.iterator().asSequence().toList() }
```

Comment:
To catchup with recent changes, this should now have `.sortedBy { it.scannerName }`.